### PR TITLE
CCTRI-1320: Upgrade Serverless Relay documentation 

### PR DESCRIPTION
**Pull request by AlinaBlahun**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,114 +1,303 @@
...
+
+- `ip`
+- `ipv6`
+- `domain`
+
+### JWT Payload Structure
+
+```json
+{
+  "key": "<Farsight_DNSDB_API_KEY>"
```
Comment (gdavoian) [inline README.md:258]: Please convert the placeholder to the upper case, i.e. "FARSIGHT_DNSDB_API_KEY" :)

File: module_settings.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Farsight DNSDB",
+  "jwt": {
+   "key": "Farsight DNSDB API KEY"
```
Comment (gdavoian) [inline module_settings.json:4]: "... API Key"

File: requirements.txt [hunk 3]
```diff
@@ -1,4 +1,5 @@
-Authlib==0.14.1
-Flask==1.1.1
-marshmallow==3.5.1
-requests==2.23.0
\ No newline at end of file
+Authlib==0.14.3
+Flask==1.1.2
+marshmallow==3.6.1
+requests==2.23.0
```
Comment (gdavoian) [inline requirements.txt:4]: Please upgrade this to the latest version: 2.24.0

**Comment by gdavoian** [comment]
Please also remove `deploy-requirements.txt` and `use-requirements.txt` from the project.

File: README.md (hunk 1 above)
Comment (AlinaBlahun) [reply README.md]: Done, thanks!

File: module_settings.json (hunk 2 above)
Comment (AlinaBlahun) [reply module_settings.json]: Done

File: requirements.txt (hunk 3 above)
Comment (AlinaBlahun) [reply requirements.txt]: Done

**Pull request merged**